nautobot/circuits/models.py

```diff
@@ -282,7 +282,6 @@ def to_csv(self):
     "graphql",
     "locations",
     "relationships",
-    "statuses",
```

Comment:
Should we get a backlog item to add this as intended?

Reply:
As in add a `status` field to CircuitTermination model? I don't see a need to do so at this time.